src/components/PublicBeatPlayButton.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { useState } from 'react';
+import { MusicContext } from '../util/contexts/music';
+import { makeStyles } from '@material-ui/core/styles';
+
+interface PublicBeatPlayButtonProps {
+  isPlaying: boolean,
+  togglePlayPauseButon: any
+}
+
+const useStyles = makeStyles(() => ({
+  playButton: {
+    cursor: 'pointer'
+  }
+}));
+
+const PublicBeatPlayButton: React.FC<PublicBeatPlayButtonProps> = ({...props}) => {
+  const [isCardPlaying, setIsCardPlaying] = useState(false);
+  const musicProvider = React.useContext(MusicContext);
+
+  const classes = useStyles();
+
+  const playOrPause = (event: React.MouseEvent<HTMLImageElement>) => {
+    setIsCardPlaying(!props.isPlaying);
+    props.togglePlayPauseButon();
+  };
+
+  let button;
+
+  if (isCardPlaying) {
+    button = (
+      <img className={classes.playButton} alt='' src='images/pauseButton.png' onClick={playOrPause}></img>
+    );
+  } else {
+    button = (
+      <img className={classes.playButton} alt='' src='images/playButton.png' onClick={playOrPause}></img>
+    );
+  }
+
+  return (
+    <>
+    {button}
+    </>
```

Comment:
What's `<>`?

Reply:
It is called Fragment, sometimes you don't want to have so many nested components like <div>, it's hard to style or debug or going through the whole nested tree to find our components, so React has this Fragment <></> to be able to render a component without a parent tag.